telegramBot.js

```diff
@@ -0,0 +1,200 @@
+            } else { 
+                ctx.reply('Bot uninitialized for this Chat. Run the /init command to initialize.')
+            }
+        })
+
+        this.bot.command('help', (ctx) =>{
+            const helpText = `You can control me by sending these commands:\n
+                                \t/init - initialize bot\n
+                                \t/status - check status of bot\n
+                                \t/gm <platform> <message> - sends text message to chosen platform\n`
+
+            ctx.reply(helpText)
+        })
+
+        this.bot.command('quit', (ctx) => {
+            ctx.leaveChat()
+        })
+        
+        this.bot.command('gm', async (ctx) => {
+            if(this.initialzied) {
+
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+                
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+        
+        this.bot.command('fs', async (ctx) => {
+            if(this.initialzied)  {
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+            
+                const fileURL = "http://i.imgur.com/5rOhtdL.png"
+                this.sendFile(fileURL)
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+
+        this.bot.command('fs_r', async (ctx) =>{
+            if(this.initialzied)  {
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+            
+                const fileURL = 'https://homepages.cae.wisc.edu/~ece533/images/airplane.png'
+                this.sendFileRemote(fileURL)
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+
+        this.bot.use((ctx) => {
```

Comment:
not sure if you need the middleware to cache the `chatID ` and `chatTitle `, since you could get it from the context object

Reply:
Keeping this ensure that the bot is synced to the chat. It's just a backup